### ISSUE TITLE
Improve error message when using wrong syntax to import a type

### ISSUE THREAD

**Issue by xhh** [gleam-experiments/snag#3]
Code:

```gleam
import gleam/io
import gleam/string.{inspect}
import snag.{Result, Snag}

pub fn main() {
  let r: snag.Snag = snag.new("hello")
  io.println(inspect(r))
}
```

Error:

```
Unknown module field

Did you mean `error`?

The module `snag` does not have a `Result` value.
```

OS: Windows 11

**Comment by lpil** [comment]
Hello! The syntax is `import snag.{type Result}`

**Comment by lpil** [comment]
Let's improve this error message.

**Comment by xhh** [comment]
Ah, thanks!
Another issue of mine can be closed as well then.

**Comment by bruse** [comment]
I think this is fixed in Gleam v1.2.0 @lpil ?

**Comment by lpil** [comment]
Yes, thank you
